features/networking/multus.feature

```diff
@@ -1263,3 +1263,28 @@ Feature: Multus-CNI related scenarios
       | /usr/sbin/ip | route         |
     Then the output should contain:
       | 192.168.10.0                 |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-30054
+  @admin
+  Scenario: Multus namespaceIsolation should allow references to CRD in the default namespace
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    # Create the net-attach-def via cluster admin
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/NetworkAttachmentDefinitions/whereabouts-macvlan.yaml" replacing paths:
+      | ["metadata"]["namespace"] | default                                                                                                                         |
+      | ["spec"]["config"]        | '{ "cniVersion": "0.3.0", "type": "macvlan", "mode": "bridge", "ipam": { "type": "whereabouts", "range": "192.168.22.100/24"} }'|
+    Then the step should succeed
+
+    #Cleanup created net-attach-def from default namespaces
+    And admin ensures "macvlan-bridge-whereabouts" network_attachment_definition is deleted from the "default" project after scenario
```

Comment:
Generally,  this step should be before created line 1274

Reply:
@zhaozhanqi Actually, the cleanup steps in our existing multus scripts always be created after the the steps of creating net-attach-def, check this: https://github.com/openshift/verification-tests/blob/master/features/networking/multus.feature#L537 to L549. Thanks!